control_plane/roles/control_plane_monitoring/files/promtail_config.yml

```diff
@@ -0,0 +1,138 @@
+server:
+  http_listen_port: 9080
```

Comment:
Can we have this in variable?

Reply:
This file is placed in files folder, variable gets applied to only tasks file. This is a configuration file for promtail.